pkgs/stdenv/generic/default.nix

```diff
@@ -223,7 +223,7 @@ let
         # include it in the result, it *is* available to nix-env for queries.
         meta = { }
             # If the packager hasn't specified `outputsToInstall`, choose a default,
-            # namely `p.bin or p.out or p`;
+            # namely `lib.getBin p`;
```

Comment:
- I believe the logic is the same. The default output is the first one in the `outputs` list.
- Especially if the logic wasn't the same, the comment should correspond to the implementation. Otherwise it's confusing.